pkg/duck/sinks.go

```diff
@@ -58,6 +60,26 @@ func NewSinkReconciler(opt reconciler.Options, callback func(string)) *SinkRecon
 	return ret
 }
 
+// NewSinkReconciler creates and initializes a new SinkReconciler
```

Comment:
So, would be great to have the better comments here in this file in general because just looking at this it's not obvious what a Sink is or a SinkReconciler, etc.

Reply:
I don't know who added it but I 100% don't understand why it is called a sync reconciler.

Reply:
Tracking: https://github.com/knative/eventing/issues/1370